source/shared/cpp/ObjectModel/Enums.h

```diff
@@ -156,7 +157,7 @@ enum class TextWeight {
     Bolder
 };
 
-enum class TextColor {
+enum class AdaptiveColor {
```

Comment:
>AdaptiveColor [](start = 11, length = 13)

We don't use Adaptive prefix elsewhere in the shared model. Should we change this just to Color in this layer?

Reply:
Yeah, i did this because i was conflicting with Windows::UI::Color in the renderer layer, and "color" seemed particularly broad compared to some of the other names.  That said it is inconsistent with what else we're doing at this layer, so i'll take a swing at a rename and handle the conflicts with more namespaces.

---
In reply to: [138413824](https://github.com/Microsoft/AdaptiveCards/pull/620#discussion_r138413824) [](ancestors = 138413824)

Reply:
I may be overthinking it, but my main concern comes from Android. Swig will generate the object model, and this will be the only instance of an Adaptive prefix there.

---
In reply to: [138680427](https://github.com/Microsoft/AdaptiveCards/pull/620#discussion_r138680427) [](ancestors = 138680427,138413824)